src/proto/message.pro

```diff
@@ -11,9 +11,17 @@ int emsg_not_now(void);
 void ignore_error_for_testing(char_u *error);
 void do_perror(char *msg);
 int emsg(char *s);
-int semsg(const char *s, ...);
+int semsg(const char *s, ...)
+#ifdef USE_PRINTF_FORMAT_ATTRIBUTE
+   __attribute__((format(printf, 1, 2)))
+#endif
+;
```

Comment:
The `*.pro` files are automatically generated by the `cproto` command.
The changes to `*.pro` will be overwritten.

Reply:
OK, I did not know that. I'll remove them for now, but I
think it's a shame not not have those `__attribute__`
as they obviously help to detect incorrect format.